src/core_plugins/input_control_vis/public/control/filter_manager/filter_manager.js

```diff
@@ -52,6 +52,10 @@ export class FilterManager {
     throw new Error('Must implement getValueFromFilterBar.');
   }
 
+  setUnsetValue(unsetValue) {
```

Comment:
How about: `setDefaultValue` or something `setUnsetValue` really is breaking my brain when I read it!

Reply:
I don't think default is the right word because its not a useable value - just a value indicating an empty state. I think keeping  it called unsetValue makes the most sense since it makes everything consistent.

Reply:
This is still confusing me also, I thought it was a defaultValue. Is this here primarily for the range control?  Does "unset" values make sense in the context of the drop down control?  The only control which seems to have an "unset" value is range control when I guess the input field is cleared out, and we need to give the range a value to use?   

But now that I say that... that still seems like a default value to me.  Isn't a default value what's used when there is an empty state?  

Or does the "unset" value change in the range slider because someone could update the range from 1 - 2 to be 5 - 10, then clear out one of the end input values, but the unset value is still 5 - 10?

I think maybe this concept (if I'm understanding it right, I could still be totally off) is very specific to the way we have built our UI for range sliders and the logic shouldn't be generalized this far up.